source/_patterns/02-organisms/50-borrower-dashboard/06-auto-repayment-terms.mustache

```diff
@@ -0,0 +1,23 @@
+<div class="small-12 large-10 columns">
+	<h1 class="impact">
+		{{headerText}}
+	</h1>
+	{{#terms}}
```

Comment:
i haven't seen lists of values used this way.  I would have expected

```
{{#terms}}
  <div>
   <h2>{{topic}}</h2>
   <p>{{answer}}</p>
  </div>
{{/terms}}
```

e.g. why does the template have to know how many topics there are?  and in the json, you could just have a list of {"topic":"a","answer":"b"} structures, with the keys all named the same thing.